stacks-signer/src/signerdb.rs

```diff
@@ -17,14 +17,58 @@
 use std::path::Path;
 
 use blockstack_lib::util_lib::db::{
-    query_row, sqlite_open, table_exists, u64_to_sql, Error as DBError,
+    query_row, sqlite_open, table_exists, u64_to_sql, Error as DBError, FromRow,
 };
-use rusqlite::{params, Connection, Error as SqliteError, OpenFlags, NO_PARAMS};
+use rusqlite::{params, Connection, Error as SqliteError, OpenFlags, Row, NO_PARAMS};
+use serde::Serialize;
+use serde_json::Value;
 use slog::slog_debug;
 use stacks_common::debug;
 use stacks_common::util::hash::Sha512Trunc256Sum;
+use wsts::net::Packet;
+use wsts::state_machine::coordinator::State as CoordinatorState;
+use wsts::state_machine::signer::State as SignerState;
+
+use crate::signer::{coordinator_state_to_string, signer_state_to_string, BlockInfo};
+
+/// The Outbound Message info
+#[derive(Debug, Serialize, Clone, PartialEq)]
+pub struct OutboundMessageInfo {
+    /// The messages sent to listening parties
+    pub outbound_messages: Vec<Packet>,
+    /// the WSTS Coordinator state at the time of sending the outbound messages
+    pub coordinator_state: String,
```

Comment:
Does this have to be a `String`?  Why can't it be `CoordinatorState`?  Same below for `signer_state`